lib/cdo/env.rb

```diff
@@ -0,0 +1,19 @@
+module EnvUtils
+
+  SENSITIVE_KEYWORDS = %w[KEY PASSWORD]
+  OBFUSCATION = '(HIDDEN)'
+
+  #
+  # ENV.redact_sensitive_values - returns a hash of all environment variables with sensitive* values obfuscated
+  #   * an env var is considered sensitive when its name contains any SENSITIVE_KEYWORDS
+  #
+  def ENV.redact_sensitive_values
```

Comment:
I still like your suggestion as it's more clear. I'll rename to with_sensitive_values_redacted.